pallets/runtime-interfaces/src/lib.rs

```diff
@@ -48,6 +51,27 @@ const MINER_DEFAULT: &str = "Eth:0x0000000000000000000000000000000000000000";
 const ETH_RPC_URL_DEFAULT: &str = "https://ropsten-eth.compound.finance";
 const OPF_URL_DEFAULT: &str = "https://prices.compound.finance/coinbase";
 
+fn validator_config_interface_get_internal(key: &str) -> Option<String> {
+    // check env override
+    if let Ok(eth_key_id_from_env) = std::env::var(key) {
```

Comment:
wouldn't be eth_key_id necessarily here right? just whatever was requested from the key?

Reply:
yeah good catch, this just came out of the refactor

Reply:
thanks 03096ee